packages/patternfly-4/react-core/src/components/Title/Title.test.tsx

```diff
@@ -5,7 +5,7 @@ import { Title, TitleSize, TitleLevel } from '.';
 Object.values(TitleSize).forEach(size => {
   test(`${size} Title`, () => {
     const view = shallow(
-      <Title size={size} headingLevel={TitleLevel.h1}>
+      <Title size={size as 'xs' | 'sm' | 'md' | 'lg' | 'xl' | '2xl' | '3xl' | '4xl'} headingLevel={TitleLevel.h1}>
```

Comment:
does the `Title.h1` work here?  Or do we have to use the string?

Reply:
Yes, it still works. :)

Reply:
Okay, butwhy are we changing this as part of the Grid conversion?

Reply:
I was running into some errors with the tests, and this was my workaround at the time. I think I found a way that we can keep the previous code.